src/renderer/i18n/locales/en-US.js

```diff
@@ -883,7 +894,11 @@ export default {
     VOTE_DELEGATE: 'Vote Delegate {delegate}',
     UNVOTE_DELEGATE: 'Unvote Delegate {delegate}',
     VOTED_FOR: 'You voted for delegate {delegate}',
-    WALLET_VOTED_FOR: 'This wallet voted for delegate {delegate}'
+    WALLET_VOTED_FOR: 'This wallet voted for delegate {delegate}',
+    NO_VOTE: 'You haven\'t voted yet, click on the button to the right to select a delegate',
```

Comment:
Maybe "This wallet is not voting. Click on any delegate or search it to vote"?

Reply:
Changed to `This wallet is not voting - use the search or click on any delegate to cast your vote`, does that sound ok?

Reply:
Uhm, I'm not completely sure; users could think that clicking on a delegate would cast their vote directly.

Something like this "This wallet is not voting. To view more information and vote, use the search or click on any delegate"?

Reply:
```suggestion
    NO_VOTE: 'This wallet is not voting. To view more information and vote, use the search or click on any delegate',
```